### ISSUE TITLE
Building Errors when compiling solution

### ISSUE THREAD

**Issue by likemike91** [Squirrel/Squirrel.Windows#850]
Hi,

when I clone the repository and try to build the solution with Release Configuration I always get the following error:

    Cannot open include file: 'WinSDKVer.h': No such file or directory

and

    The command "cd "C:\Projects\Squirrel\src\Update\bin\Release\"
    "C:\Projects\Squirrel\packages\ILRepack.1.26.0\tools\ILRepack.exe" /internalize /out:Update-Mono.exe.tmp Update-Mono.exe WpfAnimatedGif.dll ICSharpCode.SharpZipLib.dll    Microsoft.Web.XmlTransform.dll Splat.dll DeltaCompressionDotNet.dll   DeltaCompressionDotNet.MsDelta.dll Squirrel.dll NuGet.Squirrel.dll Mono.Cecil.dll
    del "Update-Mono.exe"
    ren "Update-Mono.exe.tmp" "Update-Mono.exe"" exited with code 1.	Update-Mono			

When I build it with Debug Configuration everything works correct?

My Setup:

- Windows 10 Enterprise x64
- Visual Studio 2015.3 Professional
- Windows 10 Anniversary SDK

Can you tell me what's the error here?

**Comment by mercatochris** [comment]
I'm not a C++ developer by trade, however I did manage to get it to build. 

I retargeted the project to SDK 8,1 version - make sure you install the SDK first. I used 8.1 target platform, and Visual Studio 2015 (v140.) 

Once I got it building I was hitting problems in the C# portion of the project which was during the releasify process (looks like issues with the nugget compression) which I still haven't overcome yet. 

Its a shame because I think the approach is interesting, but ATM there's a lot that needs work on especially if you are working on a recent .net version > 4.5.
